perf_test/Benchmark_Context.hpp

```diff
@@ -95,6 +95,21 @@ inline void add_benchmark_context(bool verbose = false) {
   add_version_info();
 }
 
+struct Params {
+  static std::map<std::string, int> params_;
+
+  static void parse_inputs(int argc, char** argv) {
+    for (int i = 1; i < argc; i += 2) {
+      params_[argv[i]] = std::atoi(argv[i + 1]);
```

Comment:
How do you know every parameter will always be an `int` and not a `bool`, a string or a floating point values?
Maybe this should be called `parse_int_inputs` and additional version should be added for other types?